src/content/docs/accounts/original-accounts-billing/product-based-pricing/overview-data-retention-components.mdx

```diff
@@ -502,7 +502,7 @@ Select a New Relic product to see details about its data retention:
       </tbody>
     </table>
 
-    In addition, Infrastructure downsamples your data on the fly, as it's generated. All Infrastructure metric data types (including [On-Host Integrations](/docs/integrations/host-integrations/getting-started/introduction-host-integrations) metrics) will display different granularity depending on the age of the data and the size of the time window. The following table illustrates when different downsampled buckets will be used, both in the Infrastructure UI and for queries:
+    In addition, infrastructure monitoring downsamples your data on the fly, as it's generated. All Infrastructure metric data types (including [On-Host Integrations](/docs/integrations/host-integrations/getting-started/introduction-host-integrations) metrics) will display different granularity depending on the age of the data and the size of the time window. The following table illustrates when different downsampled buckets will be used, both in the Infrastructure UI and for queries:
```

Comment:
I feel like "on the fly, as it's generated" is a bit redundant and can probably remove the on the fly part, but that's just a small thing.